### PR TITLE
Added Calicoctl to deployment of Master Nodes when Calico is used

### PR DESCRIPTION
**Pull request by VincentS**
Added Task to `Calico Master Role` that  installs calicoctl on the Master Nodes of the OpenShift cluster when Calico is used.

### CODE REVIEW

File: roles/calico_master/tasks/main.yml [hunk 1]
```diff
@@ -39,3 +39,10 @@
     resource_kind: scc
     resource_name: privileged
     state: present
+
+- name: Download Calicoctl
+  become: yes
+  get_url:
+    url: https://github.com/projectcalico/calicoctl/releases/download/v1.1.3/calicoctl
```
Comment (ingvagabund) [inline roles/calico_master/tasks/main.yml]: Just a nit, but can you move the hardcoded url into defaults/main.yml?

**Comment by VincentS** [comment]
Removed hardcoded URLs and rebased patch.

**Comment by ingvagabund** [comment]
looks good, can you squash the commits?

**Comment by ingvagabund** [comment]
aos-ci-test

**Comment by ingvagabund** [comment]
[merge]

**Comment by ingvagabund** [comment]
@VincentS please ignore the merge failure, once squashed, I will re-run the CI and merge

**Comment by VincentS** [comment]
@ingvagabund : I squashed the commits and rebased it.

**Comment by ingvagabund** [comment]
> @ingvagabund : I squashed the commits and rebased it.

Don't see any change. Maybe 'git push --force'?

**Comment by ingvagabund** [comment]
aos-ci-test

**Comment by ingvagabund** [comment]
aos-ci-test

**Pull request merged**